src/styles/_catalog.scss

```diff
@@ -19,4 +24,16 @@
   &__card:not(:nth-child(4n)) {
```

Comment:
You can use gap property in the container, instead of nth-child.

Reply:
Agree, you can remove `:nth-child`

Reply:
fixed